apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -92,12 +92,23 @@ const SendMessageToSupportService = new GQLCustomSchema('SendMessageToSupportSer
 
                 const residents = await Resident.getAll(context, { user: { id: user.id } })
 
-                const organizationsIds = residents.reduce((accumulatedData, resident) => {
+                const organizationsIds = []
+                const residentsExtraInfo = []
+                for (let resident of residents) {
```

Comment:
you can use const as you do not change resident later

Reply:
fixed